### PR TITLE
Task#877 Updated bash file to run all bathymetry steps.

### PR DESCRIPTION
**Pull request by rmcapote**
Added test data for testing the scripts.

### CODE REVIEW

File: src/LNEC_SAR_Bathymetry/Run_SAR_Bathymetry.sh [hunk 1]
```diff
@@ -24,17 +24,23 @@ python SAR_Tiling.py -a Config_Image.ini -i Aveiro_20170131T183449_20170131T1835
...
 
 
 ##*****************************************************************
 ## 3) Point Subdivision (Deep/shallow water vs computation points)
 ##*****************************************************************
-python SAR_GridPoints_Subdivision.py -a Config_Inversion.ini -l 0 -T 16.6 -m direct -w linear -v
+fun3 () {
+	local filename=$1
+	fname=$(echo $filename| cut -d'/' -f 4)
+	SAR_GridPoints_Subdivision/run -i $fname -a Config_Inversion.ini -l 0 -T 16.6 -m direct -w linear -o $fname.sub  -v
```
Comment (rasrocha) [inline src/LNEC_SAR_Bathymetry/Run_SAR_Bathymetry.sh]: Perhaps you should use braces "{}" in variables where you append something to them. For example, $fname.sub could be changed to ${fname}.sub

File: src/LNEC_SAR_Bathymetry/Run_SAR_Bathymetry.sh [hunk 2]
```diff
@@ -46,20 +52,21 @@ python SAR_GridPoints_Subdivision.py -a Config_Inversion.ini -l 0 -T 16.6 -m dir
 fun2 () {
 	local filename=$1
 	fname=$(echo $filename| cut -d'/' -f 4)
-	python SAR_Inversion.py -i $fname -v
+	SAR_Inversion/run -i $fname -o $fname.inv -v
```
Comment (rasrocha) [inline src/LNEC_SAR_Bathymetry/Run_SAR_Bathymetry.sh]: Same as my previous comment.

File: src/LNEC_SAR_Bathymetry/Run_SAR_Bathymetry_wings.sh [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+wait
+#fi
+
+##*****************************************
+## 5) Post-Processing
+##*****************************************
+##merge all information for post-processing
+dirlist=`ls Spectrum*.inv`
+sudo /var/lib/tomcat8/.wings/storage/users/admin/coresyf/code/library/sarPostprocessing/run -i $dirlist -o final_data.txt -v
+
```
Comment (rasrocha) [inline src/LNEC_SAR_Bathymetry/Run_SAR_Bathymetry_wings.sh:72]: See my comments on the other file for variables where you append something (ex: $fname.inv)

**Review by rasrocha — CHANGES_REQUESTED** [review]

File: src/LNEC_SAR_Bathymetry/Run_SAR_Bathymetry.sh (hunk 1 above)
Comment (rmcapote) [reply src/LNEC_SAR_Bathymetry/Run_SAR_Bathymetry.sh]: Done!

File: src/LNEC_SAR_Bathymetry/Run_SAR_Bathymetry.sh (hunk 2 above)
Comment (rmcapote) [reply src/LNEC_SAR_Bathymetry/Run_SAR_Bathymetry.sh]: Done!

**Pull request merged**